### PR TITLE
Unify 'loose' option for all babelify transforms

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Comment by gkatsev** [comment]
Should help out with #2335

File: grunt.js [hunk 1]
```diff
@@ -235,7 +235,7 @@ module.exports = function(grunt) {
           transform: [
             require('babelify').configure({
               sourceMapRelative: './src/js',
-              loose: 'all'
+              loose: ['all']
```
Comment (heff) [inline grunt.js]: ...really?

Comment (gkatsev) [inline grunt.js]: I don't think this was actually an issue. It seems to work just fine. However, it seems to expect an array, so, I've changed it to be that.

**Comment by heff** [comment]
If you don't mind, I'll just change this in #2254 to avoid conflicts.

**Comment by gkatsev** [comment]
I'm ok with that.

**Pull request closed without merging**